docs/sources/reference/commandline/cli.md

```diff
@@ -1443,10 +1443,9 @@ NOTE: Docker will warn you if any containers exist that are using these untagged
 
       -c, --change=[]     Apply specified Dockerfile instructions while importing the image
 
-URLs must start with `http` and point to a single file archive (.tar,
-.tar.gz, .tgz, .bzip, .tar.xz, or .txz) containing a root filesystem. If
-you would like to import from a local directory or archive, you can use
-the `-` parameter to take the data from `STDIN`.
+You can import a single file archive (.tar, .tar.gz, .tgz, .bzip, .tar.xz, or .txz) containing a root filesystem from a local file relative to the docker client.
```

Comment:
This sentence is very difficult to parse.  There are too many ideas embedded in the chained prepositions. Break them out.

---> https://gist.github.com/moxiegirl/51e5d83f7d45915879b8

You can specify a `URL` or `-` (dash) to take data directly from STDIN. The `URL` can point to an archive (.tar, .tar.gz, .tgz, .bzip, .tar.xz, or .txz) containing a fileystem or to an individual file on the Docker host.  If you specify an archive, Docker untars it in the container relative to the `/` (root). If you specify an individual file, you must specify the full path within the host. To import from a remote location, specify a `URI` that beings with the `http://` protocol.